src/app/components/ThemeProvider/fontSizes.ts

```diff
@@ -1,13 +1,17 @@
 import pixelsToRem from '../../utilities/pixelsToRem';
-import { GROUP_B_MIN_WIDTH, GROUP_D_MIN_WIDTH } from './fontMediaQueries';
+import {
+  GROUP_B_MIN_WIDTH,
+  GROUP_B_ONLY,
+  GROUP_D_MIN_WIDTH,
+} from './fontMediaQueries';
 import { GelFontSize, TypographyScript } from '../../models/types/theming';
 
 const getGelFontSizeStyles =
   (size: GelFontSize) => (script: TypographyScript) => ({
     fontSize: `${pixelsToRem(script[size].groupA.fontSize)}rem`,
     lineHeight: `${pixelsToRem(script[size].groupA.lineHeight)}rem`,
 
-    [GROUP_B_MIN_WIDTH]: {
+    [GROUP_B_ONLY]: {
```

Comment:
This is to replicate the functionality in https://github.com/bbc/simorgh/blob/latest/src/app/legacy/psammead/gel-foundations/src/getTypeSizes.js#L25